openc3/openc3.gemspec

```diff
@@ -97,6 +97,7 @@ spec = Gem::Specification.new do |s|
   s.add_runtime_dependency 'httpclient', '~> 2.8'
   # faraday includes faraday-net_http as the default adapter
   s.add_runtime_dependency 'faraday',   '~> 2.7'
+  s.add_runtime_dependency 'faraday-multipart',   '~> 1.0'
```

Comment:
I don't see us using this anywhere. According to the [docs](https://github.com/lostisland/faraday-multipart) you need to add it to the connection:
```
require 'faraday'
require 'faraday/multipart'

conn = Faraday.new(...) do |f|
  f.request :multipart, **options
  # ...
end
```